src/main/java/com/coinbase/exchange/api/exchange/GdaxExchangeImpl.java

```diff
@@ -160,4 +160,15 @@ private void curlRequest(String method, String jsonBody, HttpHeaders headers, St
         curlTest += "-X " + method + " " + getBaseUrl() + resource;
         log.debug(curlTest);
     }
+
+    private String toJson(Object object) {
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Spring provides an object mapper by default so we can autowire one into the constructor and utilise it here.